lua/gitblame/init.lua

```diff
@@ -99,13 +100,13 @@ local function process_blame_output(blames, filepath, lines)
                 info.author = author
             elseif line:match("^author%-time ") then
                 local text = line:gsub("^author%-time ", "")
-                info.date = text
```

Comment:
`info.date` is typed as `osdate` but the and used as an `integer` but the actual blame looks like this:

```
author-mail <sam@bossley.xyz>
author-time 1696555401
author-tz -0400
```

So it's technically a string representation of an integer timestamp. Just to be save, we can convert the string to a number and fallback to `os.time()`

Reply:
Well, the problem is that `osdate` is the wrong type because it's internally typed as a table like this

```
{year = 1998, month = 9, day = 16, yday = 259, wday = 4,
     hour = 23, min = 48, sec = 10, isdst = false}
```

but the actual time is an integer representing a date such as

```
1696555401
```

I agree though, it is a bit misleading to see `integer` when it's a numerical representation of a date. I could do something like this according to the docs?

```lua
---@alias timestamp integer unix epoch representation of time

-- later in code...
---@type date timestamp
```

but luals still displays it as an integer. What are your thoughts?

Reply:
Oh, okay! Yeah, I think a type alias could be great here :) 
> but luals still displays it as an integer

I think it's fine. It may get upgraded over time to display the type alias itself, too. And we'll refer to it as a timestamp in the code, so that's already good!

Reply:
Sounds good! Done in https://github.com/f-person/git-blame.nvim/pull/120/commits/38dfe4fdf3d43629387f2edb40df722d8194e2fe